sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6462 @@
-          "displayMask": "",
-          "truncateMultipleSpaces": false
+            "persistent": false,
+            "clearOnHide": true,
+            "refreshOn": "",
+            "redrawOn": "",
+            "dataGridLabel": false,
+            "labelPosition": "top",
+            "description": "",
+            "errorLabel": "",
+            "tooltip": "",
+            "hideLabel": false,
+            "tabindex": "",
+            "disabled": false,
+            "autofocus": false,
+            "dbIndex": false,
+            "customDefaultValue": "",
+            "calculateValue": "",
+            "calculateServer": false,
+            "widget": null,
+            "validateOn": "change",
+            "validate": {
+                "required": false,
+                "custom": "",
+                "customPrivate": false,
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false
+            },
+            "allowCalculateOverride": false,
+            "encrypted": false,
+            "showCharCount": false,
+            "showWordCount": false,
+            "allowMultipleMasks": false,
+            "id": "ee9wfci",
+            "addons": []
         },
         {
-          "label": "Program description",
-          "autoExpand": false,
-          "showCharCount": true,
-          "tableView": true,
-          "validate": {
-            "customMessage": "Too Long",
-            "maxLength": 500,
-            "required": false,
-            "custom": "",
-            "customPrivate": false,
-            "strictDateValidation": false,
+            "title": "Panel",
+            "theme": "default",
+            "tooltip": "",
+            "customClass": "",
+            "collapsible": false,
+            "hidden": false,
+            "hideLabel": true,
+            "disabled": false,
+            "modalEdit": false,
+            "key": "panel",
+            "tags": [],
+            "properties": {},
+            "customConditional": "",
+            "conditional": {
+                "json": "",
+                "show": null,
+                "when": null,
+                "eq": ""
+            },
+            "logic": [],
+            "attributes": {},
+            "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+            },
+            "type": "panel",
+            "label": "Panel",
+            "breadcrumb": "default",
+            "tabindex": "",
+            "input": false,
+            "tableView": false,
+            "components": [
+                {
+                    "label": "Level Of Study Codes",
+                    "labelWidth": "",
+                    "labelMargin": "",
+                    "customClass": "",
+                    "modalEdit": false,
+                    "defaultValue": "levelOfStudyCodes",
```

Comment:
why default value id `levelOfStudyCodes`

Reply:
Mistake, I will remove it.

Reply:
Removed.